crypto/bn254/signer.go

```diff
@@ -0,0 +1,9 @@
+package bn254
```

Comment:
why does this all live in bn254? shouldnt signer interface be independent of signing scheme?

Reply:
hmm you are right actually - although I just want the signer to be for BLS signer - the ethereum signer will have different interfaces. We can have this signer support bn254 or bls12381. would that be okay?